<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simple Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(50, 50, 50)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="DisplayLine">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>401</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: gray;
	border: 1px solid gray;
	color: rgb(255, 255, 255)
}</string>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 255, 255);
	border: 1px solid gray;
	padding: 3px;
}

QPushButton:pressed {
	background-color: gray;
	border:1px solid gray;
	padding: 3px;
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_8">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 255, 255);
	border: 1px solid gray;
	padding: 3px;
}

QPushButton:pressed {
	background-color: gray;
	border:1px solid gray;
	padding: 3px;
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_9">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 255, 255);
	border: 1px solid gray;
	padding: 3px;
}

QPushButton:pressed {
	background-color: gray;
	border:1px solid gray;
	padding: 3px;
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 255, 255);
	border: 1px solid gray;
	padding: 3px;
}

QPushButton:pressed {
	background-color: gray;
	border:1px solid gray;
	padding: 3px;
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 255, 255);
	border: 1px solid gray;
	padding: 3px;
}

QPushButton:pressed {
	background-color: gray;
	border:1px solid gray;
	padding: 3px;
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 255, 255);
	border: 1px solid gray;
	padding: 3px;
}

QPushButton:pressed {
	background-color: gray;
	border:1px solid gray;
	padding: 3px;
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 255, 255);
	border: 1px solid gray;
	padding: 3px;
}

QPushButton:pressed {
	background-color: gray;
	border:1px solid gray;
	padding: 3px;
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 255, 255);
	border: 1px solid gray;
	padding: 3px;
}

QPushButton:pressed {
	background-color: gray;
	border:1px solid gray;
	padding: 3px;
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>180</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 255, 255);
	border: 1px solid gray;
	padding: 3px;
}

QPushButton:pressed {
	background-color: gray;
	border:1px solid gray;
	padding: 3px;
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 255, 255);
	border: 1px solid gray;
	padding: 3px;
}

QPushButton:pressed {
	background-color: gray;
	border:1px solid gray;
	padding: 3px;
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_0">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 255, 255);
	border: 1px solid gray;
	padding: 3px;
}

QPushButton:pressed {
	background-color: gray;
	border:1px solid gray;
	padding: 3px;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Sign">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>220</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 255, 255);
	border: 1px solid gray;
	padding: 3px;
}

QPushButton:pressed {
	background-color: gray;
	border:1px solid gray;
	padding: 3px;
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Divide">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgba(151, 242, 255, 207);
	border: 1px solid gray;
	padding: 3px;
}

QPushButton:pressed {
	background-color: gray;
	border:1px solid gray;
	padding: 3px;
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Multiply">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgba(151, 242, 255, 207);
	border: 1px solid gray;
	padding: 3px;
}

QPushButton:pressed {
	background-color: gray;
	border:1px solid gray;
	padding: 3px;
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Minus">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>180</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgba(151, 242, 255, 207);
	border: 1px solid gray;
	padding: 3px;
}

QPushButton:pressed {
	background-color: gray;
	border:1px solid gray;
	padding: 3px;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Add">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>220</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgba(151, 242, 255, 207);
	border: 1px solid gray;
	padding: 3px;
}

QPushButton:pressed {
	background-color: gray;
	border:1px solid gray;
	padding: 3px;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Equals">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>220</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgba(151, 242, 255, 207);
	border: 1px solid gray;
	padding: 3px;
}

QPushButton:pressed {
	background-color: gray;
	border:1px solid gray;
	padding: 3px;
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Exponent">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>180</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgba(151, 242, 255, 207);
	border: 1px solid gray;
	padding: 3px;
}

QPushButton:pressed {
	background-color: gray;
	border:1px solid gray;
	padding: 3px;
}</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Modulo">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>140</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgba(151, 242, 255, 207);
	border: 1px solid gray;
	padding: 3px;
}

QPushButton:pressed {
	background-color: gray;
	border:1px solid gray;
	padding: 3px;
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Display_Delete">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>100</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgba(151, 242, 255, 207);
	border: 1px solid gray;
	padding: 3px;
}

QPushButton:pressed {
	background-color: gray;
	border:1px solid gray;
	padding: 3px;
}</string>
    </property>
    <property name="text">
     <string>Del</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>425</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCalculator_App">
    <property name="title">
     <string>Calculator App</string>
    </property>
   </widget>
   <addaction name="menuCalculator_App"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
